courses/fundamentals_of_ada/030_basic_types.rst

```diff
@@ -757,96 +563,6 @@ Character Types
       Control : EBCDIC := 'A';
       Nullo : EBCDIC := nul;
 
```

Comment:
Why was character type removed? Lots of good basic information, pointer to common character/string-based functions